HW02_B_Mimic_starter.md

```diff
@@ -0,0 +1,81 @@
+HW02\_B\_Graph-Mimic
+================
+Julia Shangguan
+
+``` r
+library("ggplot2")
+library("magrittr") 
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) 
+library("ggrepel")
+```
+
+## HW02 Part B
+
+### Graph 1
+
+``` r
+ggplot(diamonds, aes(x = cut, fill = clarity)) +
+  geom_bar(position = "dodge") + 
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000,fill = "black", alpha = 0.2) +
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") +
+  theme_bw() +
+  xlab("Diamond Cut") +
+  ylab("Number of Diamonds") +
+  labs(title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by\ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(hjust = 0.5)) 
+```
+
+![](HW02_B_Mimic_starter_files/figure-gfm/unnamed-chunk-1-1.png)<!-- -->
+
+### Graph 2
+
+``` r
+ggplot(iris, aes(x = Sepal.Length, y = Petal.Length, color = Species, shape = Species)) +
+  geom_point() +
+  facet_wrap(~Species, ncol = 3, scales = "free_y") +
+  xlim(4,8) +
```

Comment:
I found that not using this function actually better matched the example graph - but this is a super useful function that I did not know about!